src/browser/api/application.js

```diff
@@ -524,6 +535,21 @@ Application.run = function(identity, configUrl = '' /*, callback , errorCallback
             type: 'connected',
             uuid
         });
+
+        const parentConfigUrl = coreState.getConfigUrlByUuid(uuid);
+
+        rvmBus.registerLicenseInfo({
+            licenseKey: mainWindowOpts.licenseKey,
```

Comment:
this should be the full option set here, `mainWindowOpts` was just a bad existing variable name